src/ccl_massfunc.c

```diff
@@ -573,6 +573,12 @@ double ccl_massfunc(ccl_cosmology *cosmo, double halomass, double a, double odel
 	  ccl_cosmology_set_status_message(cosmo, "ccl_background.c: ccl_cosmology_compute_growth(): Support for the halo mass function in cosmologies with massive neutrinos is not yet implemented.\n");
 	  return NAN;
   }
+
+  if (halomass > 1e17 || halomass < 1e6){
```

Comment:
I think these should be set to `spline_params.LOGM_SPLINE_MIN` and `spline_params.LOGM_SPLINE_MAX`
@villarrealas do you agree?

Reply:
I forgot that we were making those user accessible. I agree that this change should be made before merging.

Reply:
Agree! I'll change it